cloudinit/config/cc_salt_minion.py

```diff
@@ -149,9 +154,16 @@ def handle(name: str, cfg: Config, cloud: Cloud, args: list) -> None:
     if cloud.distro.osfamily == "freebsd":
         bsd_utils.set_rc_config_value("salt_minion_enable", "YES")
```

Comment:
these two lines should be replaced with calls `distro.manage_service("enable", const.srv_name)`